tests/Unit/PointwiseFunctions/AnalyticData/GrMhd/Test_MagnetizedTovStar.cpp

```diff
@@ -83,20 +83,28 @@ SPECTRE_TEST_CASE("Unit.PointwiseFunctions.AnalyticData.GrMhd.MagTovStar",
         CHECK_ITERABLE_APPROX(get<tag>(mag_tov_values), get<tag>(tov_values));
       });
 
-  // Verify that the resulting magnetic field has (approximately) vanishing
-  // covariant divergence
-  const auto b_field =
-      get<hydro::Tags::MagneticField<DataVector, 3, Frame::Inertial>>(
-          mag_tov.variables(in_coords, tmpl::list<hydro::Tags::MagneticField<
-                                           DataVector, 3, Frame::Inertial>>{}));
-  const auto sqrt_det_spatial_metric =
-      get<gr::Tags::SqrtDetSpatialMetric<DataVector>>(mag_tov.variables(
-          in_coords, tmpl::list<gr::Tags::SqrtDetSpatialMetric<DataVector>>{}));
-  auto tilde_b = b_field;
-  for (size_t i = 0; i < 3; ++i) {
-    tilde_b.get(i) *= get(sqrt_det_spatial_metric);
-  }
+  {
```

Comment:
why indent this? Now I need to compare every line before and after... Please remove the indentation so it's easier to review

Reply:
You can hide whitespace changes on GitHub for this sort of thing. Anyway it's not important, I just removed the indentation..